documentation/milestones/milestone_template.md

```diff
@@ -15,12 +51,17 @@
 - [ ] Translations are live in PROD (if necessary)
 
 ### Proposed metrics for measuring goals/value/definition of done
+<!-- Required -->
+
+1. {Metric 1}
 
 ### Destination for live updating metrics
+<!-- Optional -->
```

Comment:
I think this should be necessary, since public metrics are such a key part of this project's approach.

Reply:
Good suggestion! Changed this to required